### DISCUSSION TITLE
Increase mining node spawns

### DISCUSSION THREAD

**Discussion by krutoj87** [chromiecraft/chromiecraft#7217] [Suggestions]
A few months back a bug made mining modes spawn with an increased rate. I actually liked that and from World chat my impression is, that other players also liked the hyperspawn of linking modes. So is it possible to bring this back?

Currently it's very hard to farm some ore, which makes leveling professions for new chars and alts harder. Especially mithril is a pain to farm. 

Actually it would probably be enough to increase spawn rates in the Vanilla zones, because in the outland it's mostly fine. Only Khorium is super rare, but I guess that's intended. But in Azeroth some ores are really hard to farm and could really use increased spawn rates.

**Comment by dimidi1** [comment]
I absolutely agree. It's extremely hard to farm nodes and they shouldn't be so scarce... Even in Outland, farming Fel Iron is a pain in the butt as it requires you to spend way more hours just for Fel Iron Bars, not to mention Fel Steel ones... And additionally, Mote of (whatever) are also a total menace... Getting from 360 BS to 375 shouldn't be as time consuming as it is, even compared to blizz like standards. For instance,  I now want to craft 4 Flamebane Gloves which require only 4 Primal Fire and 4 Primal Water, so 160 Motes in total  for each Primal Element and not each Elemental drops motes, and that means I have to kill more than double the amount if Elementals just to get those... Do not know if I explained it correctly, but ye...

**Comment by nurobzen** [comment]
It is really painful, basically no nodes available and respaw takes forever, 3 days to farm 60 mithril ores... spent 12 hours in total...
Had to go everyone on the continents to farm 12 on the whole map....
